Solved-Problems/GraphTheory/DFS.java

```diff
@@ -0,0 +1,69 @@
+import java.util.ArrayList;
+
+public class DFS {
+  // 1- --3
+  // / | \
+  // 0 | 5--6
+  // \ | /
+  // 2-- - 4
+
+  public static void main(String[] args) {
+    int vertex = 7;
+    @SuppressWarnings("unchecked") ArrayList<Edge> graph[] = new ArrayList[vertex];
```

Comment:
Please fix the usage of brackets

```suggestion
    @SuppressWarnings("unchecked") ArrayList<Edge> [] graph = new ArrayList[vertex];
```